config/locales/en.yml

```diff
@@ -71,20 +71,20 @@ en:
         forms/bulk_upload_lettings_resume/fix_choice:
           attributes:
             choice:
-              blank: You must select how would you like to fix errors
+              blank: Select how would you like to fix these errors
               inclusion: You must select one of the following options for how would like to fix errors
-        forms/bulk_upload_lettings_soft_validations_check/soft_errors_valid:
+        forms/bulk_upload_lettings_soft_validations_check/confirm_soft_errors:
           attributes:
-            soft_errors_valid:
+            confirm_soft_errors:
               blank: You must select if there are errors in these fields
-        forms/bulk_upload_sales_soft_validations_check/soft_errors_valid:
+        forms/bulk_upload_sales_soft_validations_check/confirm_soft_errors:
           attributes:
-            soft_errors_valid:
+            confirm_soft_errors:
               blank: You must select if there are errors in these fields
         forms/bulk_upload_sales_resume/fix_choice:
             attributes:
               choice:
-                blank: You must select how would you like to fix errors
+                blank: Select how would you like to fix these errors
```

Comment:
I've just seen this, but this should be how **you would** not **would you** - missed this on my PR too, have just made that change there myself